hadoop-ozone/dist/src/main/compose/test-all.sh

```diff
@@ -23,7 +23,7 @@ SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )
 ALL_RESULT_DIR="$SCRIPT_DIR/result"
 PROJECT_DIR="$SCRIPT_DIR/.."
 mkdir -p "$ALL_RESULT_DIR"
-rm "$ALL_RESULT_DIR"/* || true
+rm -rf "${ALL_RESULT_DIR:?}"/* || true
```

Comment:
We can simplify this by swapping `rm` and `mkdir`:

```suggestion
rm -fr "${ALL_RESULT_DIR}"
mkdir -p "${ALL_RESULT_DIR}"
```

`:?` is not needed, since `ALL_RESULT_DIR` is defined in line 23.

Reply:
Not able to apply the suggestion directly in Github.
```
Applying suggestions on deleted lines is currently not supported.
```

I will address the suggestion in new commit.